ETC.tsv

```diff
@@ -916,7 +916,7 @@ ETC_20150317_000915	Woodfung
 ETC_20150317_000916	The Woodfung is closely related to witchcraft.
 ETC_20150317_000917	{memo X}Red Infrorocktor
 ETC_20150317_000918	They exhibit a unique hull that is a mix between stone and iron.
-ETC_20150317_000919	Contaminated Vikaras Mage
+ETC_20150317_000919	Corrupt Vikaras Mage
```

Comment:
Corrupt can be used as an adjective. To say corrupted would just make it use its verb form (to become corrupt).